queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
```

Comment:
- [ ] Great work writing accurate queries. For this query, the `date_of_birth` should be between `'2016-01-01'` AND `'2019-12-31'`. Kindly fix it. 👍